web/src/test/auto/headless/engine/keyboard/keyboard-loading.js

```diff
@@ -4,8 +4,9 @@ import fs from 'fs';
 import { createRequire } from 'module';
 const require = createRequire(import.meta.url);
 
-import { KeyboardHarness, KeyboardInterface, KeyboardLoaderBase, MinimalKeymanGlobal, Mock } from '@keymanapp/keyboard-processor';
-import { NodeKeyboardLoader } from '@keymanapp/keyboard-processor/node-keyboard-loader';
+import { KeyboardHarness, MinimalKeymanGlobal, Mock } from 'keyman/engine/keyboard';
+import { KeyboardInterface } from 'keyman/engine/js-processor';
+import { NodeKeyboardLoader } from 'keyman/engine/keyboard/node-keyboard-loader';
 
 describe('Headless keyboard loading', function() {
```

Comment:
OK, _this_ one has nuance.

The ideal for the "Minimal harness loading" section is to lie here without referencing `KeyboardInterface` aside from within the `cannot evaluate rules` test.  (Also, I've tested locally - the reference within `successfully loads (has variable stores)` can be replaced with `KeyboardHarness` without issue.)

The "Full harness loading" section belongs under `engine/js-processor` testing, though.